src/frontend/js/components/AshleyEditor/index.tsx

```diff
@@ -35,6 +45,7 @@ interface MyEditorProps {
   target: HTMLInputElement;
   emojiConfig?: EmojiPluginConfig;
   linkPlaceholder?: string;
+  mention_users?: MentionData[];
```

Comment:
Snake case in da place ?! 🐍
```suggestion
  mentionUsers?: MentionData[];
```

Reply:
true ! lint didn't see that one but JB did ! :-)